@here/harp-mapview/lib/MapView.ts

```diff
@@ -931,6 +938,11 @@ export class MapView extends THREE.EventDispatcher {
 
         this.setupRenderer();
 
+        if (this.m_options.enableShadows === true) {
+            this.m_renderer.shadowMap.enabled = true;
+            this.m_renderer.shadowMap.type = THREE.PCFSoftShadowMap;
```

Comment:
I think we should expose the shadow map type in the MapViewOptions as well

Reply:
as discussed, will be done in later ticket, because VSM support doesn't yet work